### PR TITLE
feat(hardware): added read temperature from pressure sensor

### PR DESCRIPTION
**Pull request by caila-marashaj**
- added function in sensor_utils.py to read temperature from pressure sensor
- added case in sensors.py for pressure_temperature read

Note: rather than adding two new functions for reading from and writing to the pressure_temperature sensor, I added an optional argument to the existing ones so they can be called with `pressure_temperature`

### CODE REVIEW

File: hardware/opentrons_hardware/scripts/sensor_utils.py [hunk 1]
```diff
@@ -160,6 +160,51 @@ async def handle_capacitive_sensor(
...
+    if include_csv:
+        metadata = CSVMetaData(
+            command.pipette_serial_number,
+            command.sensor_type.name,
+            command.minutes,
+            command.auto_zero,
+            start_time.strftime("%H:%M:%S"),
+        )
+        csv = CSVFormatter.build(metadata, list(metadata.to_dict().keys()))
+    delta = timedelta(minutes=command.minutes)
```
Comment (amitlissack) [inline hardware/opentrons_hardware/scripts/sensor_utils.py]: Is there perhaps a way to create an `end_time` using `timedelta`?

Then the loop could be `while datetime.now() < end_time:`

File: hardware/opentrons_hardware/scripts/sensor_utils.py [hunk 2]
```diff
@@ -160,6 +160,51 @@ async def handle_capacitive_sensor(
...
+    start_time = datetime.now()
+    csv = None
+    pressure = mmr920C04.PressureSensor()
+    if include_csv:
+        metadata = CSVMetaData(
+            command.pipette_serial_number,
+            command.sensor_type.name,
+            command.minutes,
+            command.auto_zero,
+            start_time.strftime("%H:%M:%S"),
```
Comment (amitlissack) [inline hardware/opentrons_hardware/scripts/sensor_utils.py]: Can we make `"%H:%M:%S"` a constant and reuse it.

File: hardware/opentrons_hardware/scripts/sensor_utils.py (hunk 1 above)
Comment (caila-marashaj) [reply hardware/opentrons_hardware/scripts/sensor_utils.py]: absolutely!

Comment (Laura-Danielle) [reply hardware/opentrons_hardware/scripts/sensor_utils.py]: @caila-marashaj can you make these two changes for the other helper functions?

File: hardware/opentrons_hardware/scripts/sensor_utils.py [hunk 3]
```diff
@@ -160,6 +160,52 @@ async def handle_capacitive_sensor(
...
+    if include_csv:
+        metadata = CSVMetaData(
+            command.pipette_serial_number,
+            command.sensor_type.name,
+            command.minutes,
+            command.auto_zero,
+            start_time.strftime(hms),
+        )
+        csv = CSVFormatter.build(metadata, list(metadata.to_dict().keys()))
+    end_time = datetime.now() + timedelta(minutes=command.minutes)
```
Comment (amitlissack) [inline hardware/opentrons_hardware/scripts/sensor_utils.py]: Maybe `start_time` instead of `datetime.now()` here?

**Review by amitlissack — COMMENTED** [review]
There aren't unit or integration tests for this code. Should there be?

**Comment by caila-marashaj** [comment]
> There aren't unit or integration tests for this code. Should there be?

Yes I'm about to push some for this

**Review by Laura-Danielle — APPROVED** [review]
Nice looks good

**Pull request merged**